tests/functional/botocore/test_endpoint_rulesets.py

```diff
@@ -171,7 +171,7 @@ def iter_e2e_test_cases_that_produce(endpoints=False, errors=False):
             if endpoints and 'endpoint' in expected_object:
                 expected_endpoint = expected_object['endpoint']
                 expected_props = expected_endpoint.get('properties', {})
-                expected_authschemes = [
+                [
```

Comment:
This requires more investigation. Its still present upstream, but it is actually used to mark a test to skip:

https://github.com/boto/botocore/blob/develop/tests/functional/test_endpoint_rulesets.py#L175C17-L189

Reply:
`expected_authschemes` isn't necessary in this test because its used to skip if CRT is needed, but AWS CLI v2 always has CRT. Will remove it.

Reply:
Fixed in d612b0336328ec6e4cac2ea6dc032bd28a23468c